### ISSUE TITLE
Is there any setting to restore cursor position on close/reopen a file? Is this supported?

### ISSUE THREAD

**Issue by it6** [evandrocoan/BufferScroll#2]

**Comment by evandrocoan** [comment]
The setting `restore_scroll` should control that: https://github.com/evandrocoan/BufferScroll/blob/master/BufferScroll.sublime-settings

**Comment by it6** [comment]
So, say I open a file navigate around (using `ctrl+n, ctrl+p`) close the file leaving cursor at line 10. Next time I open the same file my cursor is on line 1 instead of line 10. I would like it to be at `line 10`. Is this possible?

**Comment by evandrocoan** [comment]
> I would like it to be at line 10. Is this possible?

Yes, this is main purpose of this package. Here is it working fine as of right now.

If it is not working for you, it should be because the package is not running correctly. What are the full contents of your Sublime Text console? (You can open the Console by going on the menu `View -> Show Console`)

**Comment by it6** [comment]
Thanks this is exactly what I wanted, I think I figured out the problem, I am using `super+w` to close the tab, then `Command+shift+t` to reopen the closed tab which doesn't restore the cursor position. Same thing when I close the tab and reopen with `Command+p` file navigator.

I hope this would support keyboard navigation.

**Comment by evandrocoan** [comment]
Thanks! I can confirm the behavior here too. When I close the view with `Ctrl+W` the scroll is not restored when I reopen it.
